app/lib/reporting/differences.rb

```diff
@@ -78,7 +78,7 @@ def initialize
       @as_of = Time.zone.today.iso8601
     end
 
-    def generate
+    def generate(only: [])
```

Comment:
This is not strictly part of the PR,
but it's handy to generate only the report we need, for example, during development.